templates/dice/roll-options-ffg.html

```diff
@@ -1,4 +1,4 @@
-<form>
+<form style="overflow-y: scroll; scrollbar-width: thin;">
```

Comment:
I'd prefer to do the style changes via our actual SCSS but not so much that I'm not gonna merge this 😂